Reorganize Overview section (#47)
stanford-rc/www.sherlock.stanford.edu: src/docs/overview/concepts.md

    For more information about data risk classifications, see the [Information
    Security Risk Classification page][url_risk].

> ## Investing into Sherlock

Reviewer: Maybe just "Investing in Sherlock?"

> ## Investing into Sherlock

Reviewer: ```suggestion
## Investing in Sherlock
```

For users who need more than casual access to a shared computing environment,

> Sherlock also offers Faculty members the possibility to invest into additional,

Reviewer: ```suggestion
Sherlock also offers Faculty members the possibility to invest in additional,
```

dedicated computing resources.

Unlike traditional clusters, Sherlock is a collaborative system where the

> majority of nodes are purchased and shared by the cluster users, known as
> *owners*. Choosing from a standard set of server configurations supported by
> SRCC staff (known as the Sherlock *catalog*), PIs can purchase their own
> servers to add to the cluster.

Reviewer: ```suggestion
majority of nodes are purchased and shared by the cluster users. When a user (typically a PI)
purchases one or more nodes, they become an *owner*. Owners can choose from a standard set of server configurations supported by SRCC staff (known as the Sherlock *catalog*) to add to the cluster.
```.

Reviewer: I think the definition of "owner" isn't clear enough - the way it's currently written it is referencing generic cluster users.

Author: Agreed, thanks for the correction!

This model also allows Sherlock owners to benefit from the scale of the cluster
by giving them access to more compute nodes than their individual purchase,

> which gives them much greater flexibility than owning a standalone cluster.

Reviewer: Could we add a line about why that is possible? E.g., "owners share a group of nodes that, when not in use by the primary purchaser of the machine, can be used by other owners."

Author: It's detailed a little bit below with "The resource scheduler configuration works like this:", but I added a sentence to clarify.

!!! info "The majority of Sherlock nodes are owners nodes"

    The vast majority of Sherlock's compute nodes have been purchased by
    individual PIs and groups, and PI purchases are the main driver behind the
    rapid expansion of the cluster, which went from 120 nodes to more than

>     1,000 nodes in less than 3 years.

Reviewer: damn

The SRCC institutes a monthly scheduled maintenance window on Sherlock, to
ensure optimal operation, avoid potential issues and prepare for future
expansions. This window will be used to make hardware repairs, software and
firmware updates, and perform general manufacturer recommended maintenance on
our environment.

As often as possible, maintenance tasks are performed in a rolling,
non-disruptive fashion, but downtimes are sometimes an unfortunate necessity to
allow disruptive operations that can't be conducted while users are working on
the system.

!!! info "Maintenance schedule"

    As often as possible, maintenances will take place on the **first Tuesday

>     of every month, from 08:00 to 12:00** (noon), and will be announced 2 weeks

Reviewer: ```suggestion
    of every month, from 08:00 to 12:00 Pacific time** (noon), and will be announced 2 weeks
```
